bitnami/postgresql/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 8.9.8
+version: 8.9.9
```

Please bump the minor version instead, since we're adding a new feature to the chart:
```suggestion
version: 8.10.0
```